core/templates/dev/head/pages/exploration_editor/loading_modal_directive.html

```diff
@@ -0,0 +1,27 @@
+<script type="text/ng-template" id="modals/loading">
+  <div class="oppia-loading-body oppia-save-publish-loading">
```

Comment:
oppia-loading-body --> oppia-loading-modal-body

Why do you need .oppia-save-publish-loading?